src/js/components/Table/StyledTable.js

```diff
@@ -22,19 +22,19 @@ export const StyledTableCell = styled.td`
   padding: 0;
   font-weight: inherit;
   text-align: inherit;
-  height: inherit;
+  height: 100%;
```

Comment:
why are we changing this?

Reply:
It's necessary for cell backgrounds to take the full row height.